gpu4pyscf/solvent/pcm.py

```diff
@@ -425,7 +432,7 @@ def _get_qsym(self, dms, with_nuc = False):
 
         return q_sym[0], q[0]
 
-    def _get_vgrids(self, dms, with_nuc = False):
+    def _get_vgrids(self, dms, with_nuc = False, ndim = 1):
```

Comment:
The ndim=1 option is a confusing. Consider to always return 2d array. Update the tddft/pcm (or any other relevant code) to access the first vector.